designs/FloatingStableAndPrerelease.md

```diff
@@ -0,0 +1,135 @@
+
+# Support pre-release packages with floating versions
+
+* Status: **In Review**
+* Author(s): [Nikolche Kolev](https://github.com/nkolev92)
+* Issue: [912](https://github.com/NuGet/Home/issues/912) Support pre-release packages with floating versions
+* Type: Feature
+
+## Problem Background
+
+Consider a PackageReference as such:
+
+```xml
+<ItemGroup>
+    <PackageReference Include="NuGet.Packaging" Version="5.*" />
+</ItemGroup>
+```
+
+Right now this will only include stable package versions.
+
+| Floating Version | Available Versions | Best matching version |
+| -----------------|--------------------|-----------------------|
+| 5.* | 5.0.0 <br> 5.1.0 <br> 5.2.0 <br> 5.3.0 <br> 5.4.0 <br> 5.5.0-preview.1 <br> 5.5.0-preview.2 | 5.4.0 |

```

Comment:
This renders funny. I think you'll want 5.4 to align vertically between `Available versions` and `Best matching versions`